packages/fxa-payments-server/src/components/Header/index.scss

```diff
@@ -0,0 +1,55 @@
+@import '../../../../fxa-content-server/app/styles/breakpoints';
+
+.fxa-settings-header {
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 8px rgba(12, 12, 13, 0.1);
+  display: flex;
+  height: 81px;
+  justify-content: space-between;
+  margin: 0 auto;
+  padding: 0 32px;
+  position: fixed;
+  top: 0;
+  white-space: nowrap;
+  width: 100%;
+  z-index: 2;
+
+  @include max-width('mobileLandscape') {
+    height: 64px;
+  }
```

Comment:
Can we set the default height to `64px` and use `@include min-width('mobileLandscape') { height: 81px` for extra mobile friendliness?